docs/source/user/file_formats.rst

```diff
@@ -129,6 +127,24 @@ browser and select the “Editor” item in the “Open With” submenu:
     <iframe src="https://www.youtube-nocookie.com/embed/b5oAoVB3Wd4?rel=0&amp;showinfo=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
   </div>
 
+JupyterLab's powerful grid viewer can load files up to the maximum string
+size in the browser. Our experience testing the grid viewer on various browsers suggests that maximum string size varies with browser.
+Below is a table that reflects the maximum size we've successfully tested the grid viewer by browser:
```

Comment:
I would delete the "Our experience testing the grid viewer ..." sentence. We said it was a browser issue in the previous sentence, and the next sentence and table very clearly indicate it depends on the browser.